docs/getting-started/install/dreamhost.md

```diff
@@ -0,0 +1,160 @@
+# Install Dokku on DreamHost Cloud Server with cloud-init
+
+Cloud-init script can be used to automate installation of Dokku on
+Dreamhost (or any other OpenStack-compatible cloud with minimal
+changes).
+
+To create a new server and install Dokku from the openstack command
+line interface:
+
+    $ openstack server create \
+       --image Ubuntu-16.04 \
```

Comment:
We currently officially support 16.04 - though it should work on 14.04. Is it possible to change this down? We can certainly update the official version in a bit - still have to work through packaging - just want to make sure we install the correct things.